jobs/STEINBAUER_Performance_Austria_Web-Developer.html

```diff
@@ -0,0 +1,51 @@
+---
+title: Full-Stack Web Developer at STEINBAUER Performance
+company: STEINBAUER Performance Austria GmbH
+url: https://www.steinbauer.cc/at/jobs/
+location: Wartberg ob der Aist near Linz / Austria
+contract: permanent
+contact:
+name: STEINBAUER Performance Austria GmbH
+email: karriere@steinbauer.cc
+phone: +43 7236 21 8 21-0
+created: !!timestamp '2017-01-11'
+tags:
+- Python
+- PHP 
+- HTML
+- CSS
+- Javascript
+- Odoo
+- Austria
+---
+# Full-Stack Web Developer
+
+# YOUR tasks
+* Full-Stack development of web applications incl. Interfaces & Web Services
+* Project supervision - Front and Backend
+* Further development of existing patforms
+* Brainstorming with the development team
+
+# YOUR PROFILE
+* University/college degree or/and appropriate training 
+* with on the job work experience
+* Experience in the development of web applications, 
+* Front/Backend- e.g. GUI solutions and CMS
+* Knowledge in Python, PHP and HTML/CSS/Javascript
+* Experience in version control (GIT) and Databanks (Postgres, MySQL)
+* Knowledge in Linux operating system (Ubuntu)
+* Good English or German conversation skills
```

Comment:
It might be worth clarifying whether your candidates need to be bilingual or not.